src/server.ts

```diff
@@ -24,13 +29,13 @@ export const createParserFailureHandler =
       request,
       response,
       logger,
-      input: request.body,
+      input: null,
```

Comment:
⚠️ behaviour changing:
no more raw body (which is failed to parse) in ResultHandler.
Use `request` in ResultHandler instead.